extension-manifest-v3/src/store/options.js

```diff
@@ -303,19 +318,36 @@ async function migrateFromV8() {
   }
 }
 
+function isEqual(a, b) {
```

Comment:
Note that it is not symmetric:
```
> isEqual({}, { foo: 'bar' })
true
> isEqual({ foo: 'bar' }, {})
false
```

Reply:
Good catch, but it is safe here - as the structure of both compared objects is always identical - they are both two versions of options. I think I'll rename the function to `isEqualOptions` to avoid confusion.